dnase/callsnpsMerge.sh

```diff
@@ -96,8 +102,14 @@ rm -f ${allreadscovfiles}
 
 unstarch ${sampleOutdir}/${name}.${mappedgenome}.coverage.allreads.starch | cut -f1-3,5 > $TMPDIR/${name}.${mappedgenome}.coverage.allreads.bedGraph
 
-#Fix problems with reads running off end of chromosome
-bedClip $TMPDIR/${name}.${mappedgenome}.coverage.allreads.bedGraph ${chromsizes} $TMPDIR/${name}.${mappedgenome}.coverage.allreads.clipped.bedGraph
+if [ ! -s "$TMPDIR/${name}.${mappedgenome}.coverage.allreads.bedGraph" ]; then
+    # The coverage.allreads.bedGraph file is empty
+    bedgraph_chrom=$(head -n 1 ${chromsizes} | cut -f1)
+    echo -e "${bedgraph_chrom}\t0\t0\t0" > $TMPDIR/${name}.${mappedgenome}.coverage.allreads.clipped.bedGraph
```

Comment:
write to to TMPDIR/${name}.${mappedgenome}.coverage.allreads.bedGraph so you can move the bedclip out of the else block. It doesn't need to be clipped, but this makes the logic clearer